decorator.rb

```diff
@@ -0,0 +1,11 @@
+require './nameable'
+class Decorator < Nameable
```

Comment:
- Please, let's also add the ` attr_accessor :nameable` here in order to access it.